popgen/dunn.go

```diff
@@ -41,19 +42,15 @@ func Dunn(b *bed.Bed, aln []fasta.Fasta, g []*Group) (float64, string) {
 func FindMaxIntra(subFa []fasta.Fasta, g *Group) int {
 	var answer int = 0
 	var faI, faJ []dna.Base
-	var faFound bool
+	var faFoundI, faFoundJ bool
 	faMap := fasta.ToMap(subFa)
 	for i := 0; i < len(g.Members); i++ {
 		for j := i + 1; j < len(g.Members); j++ {
-			faI, faFound = faMap[g.Members[i]]
-			if !faFound {
-				log.Panicf("%s not found in fasta map", g.Members[i])
+			faI, faFoundI = faMap[g.Members[i]]
+			faJ, faFoundJ = faMap[g.Members[j]]
+			if faFoundI && faFoundJ {
```

Comment:
Seems like it should be an error if one of the seq is not found?  This would mean someone asked the function to calculate Dunns for a group of sequence names, but then not all of those names had sequences.

Reply:
So we could go either way on this. Right now if some samples are missing, this function still attempts to calculate Dunn's with the remaining samples. The function that calls this one reports which samples were missing. However, for now we have been discarding any regions with any missing samples, so perhaps this is not so useful